.github/workflows/ci.yml

```diff
@@ -8,29 +8,29 @@ on:
 
 jobs:
   test:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-20.04
     env:
       MIX_ENV: test
     strategy:
       fail-fast: false
       matrix:
         include:
           - pair:
-              elixir: 1.5.3
-              otp: 19.3.6.13
+              elixir: 1.6.x
+              otp: 21.x
```

Comment:
I bumped the min support Elixir version to 1.6, as OTP 21 is the minimum supported version in Ubuntu 20.04 OS image, see https://github.com/erlef/setup-beam?tab=readme-ov-file#compatibility-between-operating-system-and-erlangotp